### ISSUE TITLE
Adjustment menu

### ISSUE THREAD

**Issue by ysk229** [apache/apisix-dashboard#2413]
Hello
Can you change the order of the dashboard menu?
Swap the order of the upstream menu with the service menu. Because the flow of apisix gateway is route->services->upstream

**Comment by Baoyuantop** [comment]
I don't think this change will help improve the user experience, let's hear more people's opinions.

**Comment by SkyeYoung** [comment]
Maybe we can support editing menu order in V3 🤔

**Comment by jwrookie** [comment]
> I don't think this change will help improve the user experience, let's hear more people's opinions.

Agree +1, BTW according to APISIX [docs](https://apisix.apache.org/docs/apisix/architecture-design/service/), Service is an abstraction of route, so `route->services->upstream ` may be inaccurate @ysk229

**Comment by bzp2010** [comment]
First clarify that services are routes set, i.e., the configuration on a service will be shared by the route unless the route overrides its configuration. Thus services are parents of routes. The service contains routes.

The customization of the order on the UI will not be considered, which is beyond the scope of #2981. But I think in the next refurbishment, service will be the concept that will be highlighted, and routes should belong to service.

If you have any questions, feel free to start a new issue!
